### ISSUE TITLE
Obscured photos and videos show-up in main screen photo list

### ISSUE THREAD

**Issue by n8fr8** [guardianproject/ObscuraCam#121]
Should we filter out photos and video generated by ObscuraCam?

**Comment by DavidMOliver** [comment]
Yes, we should.  Question: is that something that requires opening each image in the gallery and reading its metadata?  Or is there a naming convention used?  Or other?
